components/export/api/src/main/resources/ApplicationResources.properties

```diff
@@ -48,7 +49,7 @@ phenotips.export.excel.label.phenotype.meta.name=Meta
 phenotips.export.excel.label.phenotype.meta.id=ID
 
 phenotips.export.excel.label.genotype=Genotype
-phenotips.export.excel.label.genotype.name=Gene Name
+phenotips.export.excel.label.genotype.name=Gene ID
```

Comment:
@allasm It is detailed in the issue description https://phenotips.atlassian.net/browse/PT-3082